sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseFacade.java

```diff
@@ -56,13 +56,6 @@
 @Remote
 public interface CaseFacade extends CoreFacade<CaseDataDto, CaseIndexDto, CaseReferenceDto, CaseCriteria> {
 
-	List<CaseDataDto> getAllActiveCasesAfter(Date date);
```

Comment:
Removing/renaming methods in Facades is a potentially breaking change of the API.
- is there software outside this project which might use these Facade methods?
- should this be documented in the release notes?

Reply:
True, it's breaking (as almost every ticket changes the Facade interfaces), but I don't see a point to document this. It's not used outside our stack (as far as we know), the REST endpoints are stable and alternative Facade methods exist.